azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -158,12 +158,15 @@ def call_activity(self, name: str, input_: Optional[Any] = None) -> TaskBase:
         Task
             A Durable Task that completes when the called activity function completes or fails.
         """
+        if(isinstance(name, Callable)):
+            name = name._function._name
+        
```

Comment:
As we discussed offline, I recommend refining this check in a few ways
1) We don't support invoking any arbitrary callable. Instead, we need the callable of a registered Azure Functions object. So we'll need to check against a `FunctionBuilder` corresponding to an `ActivityTrigger`
2) Let's provide some error handling in case a user passes in the wrong argument type.